tests/rptest/tests/data_migrations_api_test.py

```diff
@@ -719,3 +726,51 @@ def test_migrated_topic_data_integrity(self, transfer_leadership: bool,
                                            read_blocked=False,
                                            produce_blocked=False)
                 remounted = True
+
+    @cluster(num_nodes=3, log_allow_list=MIGRATION_LOG_ALLOW_LIST)
+    def test_list_unmounted_topics(self):
+        topics = [TopicSpec(partition_count=3) for i in range(5)]
+
+        for t in topics:
+            self.client().create_topic(t)
+
+        admin = Admin(self.redpanda)
+        unmounted_response = admin.list_unmounted_topics().json()
+        assert len(
+            unmounted_response) == 0, "There should be no unmounted topics"
+
+        outbound_topics = [make_namespaced_topic(t.name) for t in topics]
+        reply = self.admin.unmount_topics(outbound_topics).json()
+        self.logger.info(f"create migration reply: {reply}")
+
+        self.logger.info('waiting for partitions be deleted')
+        self.wait_partitions_disappear(topics)
+
+        unmounted_response = admin.list_unmounted_topics().json()
+        assert len(unmounted_response["topics"]) == len(
+            topics), "There should be unmounted topics"
+
+        # Mount 3 topics based on the unmounted topics response. This ensures
+        # that the response is correct/usable.
+        inbound_topics = [
+            InboundTopic(NamespacedTopic(topic=t["topic"], namespace=t["ns"]),
```

Comment:
it doesn't look like we have any cross cluster mounting. how do i test the label/cluster uuid thing? @bashtanov

Reply:
It's not implemented yet, at least not in the API.